src/Raven.Studio/wwwroot/App/views/database/indexes/clearIndexErrorsConfirm.html

```diff
@@ -0,0 +1,29 @@
+<div class="modal-dialog" role="document">
+    <div class="modal-content">
+        <div class="modal-header">
+            <button type="button" class="close" data-bind="click: close" aria-hidden="true">
+                <i class="icon-cancel"></i>
+            </button>
+            <h4 class="modal-title" data-bind="text: title"></h4>
+        </div>
+        <div class="modal-body">
+            <p data-bind="html: subTitleHtml"></p>
+            <div data-bind="if: !clearAllIndexes() && indexesToClear().length">
+                <ul data-bind="foreach: indexesToClear()" style="max-height: 200px; overflow-y: auto;">
+                    <li class="padding padding-xs">
+                        <strong data-bind="text: $data"></strong>                   
+                    </li>
+                </ul>
+            </div>
+            <p>
+                <small class="bg-info text-info padding padding-xs">
+                    <i class="icon-warning margin-right margin-right-sm"></i>While the current errors will be cleared, the <i>index state</i> will not change.
```

Comment:
If wonder if we should be more explicit here, and write that index status of errors won't be set to normal, as it wasn't obvious for us during our call today.

Reply:
=> 
While the current indexing errors will be cleared,
an index with an 'Error state' will not be set back to 'Normal state'.